### PR TITLE
Use proper response type

### PR DESCRIPTION
**Pull request by craigsdennis**
Updates older `new Twilio\Twiml()` calls to use `new Twilio\Twiml\VoiceResponse()` and `new Twilio\Twiml\MessagingResponse()`

### CODE REVIEW

**Comment by childish-sambino** [comment]
Fixes https://github.com/twilio/twilio-php/issues/513

**Comment by childish-sambino** [comment]
Missing this one: https://github.com/TwilioDevEd/api-snippets/blob/master/guides/voice/recording-add-on-guide/use-add-on-data/example.5.x.php

File: guides/request-validation-php-lumen/example-2/example-2.5.x.php [hunk 1]
```diff
@@ -19,7 +19,7 @@ function() {
   function(Request $request) {
     $bodyLength = strlen($request->input('Body'));
 
-    $twiml = new TwiML();
+    $twiml = new VoiceResponse();
```
Comment (childish-sambino) [inline guides/request-validation-php-lumen/example-2/example-2.5.x.php]: Should be `MessagingResponse`

File: quickstart/php/sms/example-7/send-sms-during-call.5.x.php [hunk 2]
```diff
@@ -15,7 +15,7 @@
 // otherwise, consider them just another monkey
 $name = $people[$_REQUEST['From']] ?: 'Monkey';
 
-$response = new TwiML();
+$response = new VoiceResponse();
 $response->say("Hello {$name}");
 $response->message("{$name}, thanks for the call!");
```
Comment (childish-sambino) [inline quickstart/php/sms/example-7/send-sms-during-call.5.x.php]: Can't `say` and `message`. Should the second one be `sms`?

Comment (craigsdennis) [reply quickstart/php/sms/example-7/send-sms-during-call.5.x.php]: Yeah I saw that too, will fix

File: guides/request-validation-php-lumen/example-2/example-2.5.x.php (hunk 1 above)
Comment (craigsdennis) [reply guides/request-validation-php-lumen/example-2/example-2.5.x.php]: Nice catch!

**Pull request merged**